flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/benchmark/StateBackendBenchmarkUtils.java

```diff
@@ -74,6 +75,24 @@
 			case ROCKSDB:
 				rootDir = prepareDirectory(rootDirName, null);
 				return createRocksDBKeyedStateBackend(rootDir);
+			case SINGLE_KEY:
+				try {
+					return new SingleKeyStateBackend().createKeyedStateBackend(
```

Comment:
Should also be in a method like for the other backends.

Reply:
Actually I did it and forgot to force push :facepalm: